tests/test_zeus_client.py

```diff
@@ -68,23 +81,22 @@ def test_validate_dates(self):
     @mock.patch('zeus.interfaces.rest.requests')
     def test_post_empty_log(self, mock_requests):
         logs = []
+        url = '{}/logs/{}/ZeusTest'.format(FAKE_SERVER, FAKE_TOKEN)
```

Comment:
path join functions usually treat paths starting with a forward slash as coming from the root path, so if you do  join with something starting with a slash it's as if it started from the root again.

I have run into problems before using `+`, especially when it depends on url given by a user